packages/cli/src/cmds/validator/slashingProtection/export.ts

```diff
@@ -59,18 +71,38 @@ export const exportCmd: CliCommand<ExportArgs, ISlashingProtectionArgs & Account
       const genesisValidatorsRoot =
         (await metadata.getGenesisValidatorsRoot()) ?? (await getGenesisValidatorsRoot(args));
 
-      logger.verbose("Fetching the pubkeys from the slashingProtection db");
-      const pubkeys = await slashingProtection.listPubkeys();
+      logger.verbose("Fetching pubkeys from slashing protection DB");
+      const allPubkeys = await slashingProtection.listPubkeys();
+      let pubkeysToExport = allPubkeys;
+
+      if (args.pubkeys) {
+        logger.info("Filtering by given subset of pubkeys", {count: args.pubkeys.length});
+        const filteredPubkeys = [];
+
+        for (const pubkeyHex of args.pubkeys) {
+          if (!isValidatePubkeyHex(pubkeyHex)) {
+            throw new YargsError(`Invalid pubkey ${pubkeyHex}`);
+          }
+          const existingPubkey = allPubkeys.find((pubkey) => toHexString(pubkey) === pubkeyHex);
+          if (!existingPubkey) {
+            logger.warn("Pubkey not found in slashing protection DB", {pubkey: pubkeyHex});
```

Comment:
throw error vs warnings log?

I decided to just log a warning because there might be the case where the validator pubkey exists but it has not yet produced any attestations/blocks and therefore it not in slashing protection DB.